.eslintrc.yml

```diff
@@ -5,7 +5,7 @@ env:
   mocha: true
 rules:
   no-param-reassign: 0
-  max-len: [2, 82, 4, { "ignoreComments": true, "ignorePattern":  '^\s*const\s.+=\s*require\s*\(' }]
+  max-len: [2, 85, 4, { "ignoreComments": true, "ignorePattern":  '^\s*const\s.+=\s*require\s*\(' }]
```

Comment:
Why don't we use 80?

Reply:
Why do we have an ignore pattern?

Reply:
The ignore pattern is for some of the longer `require` assignment expressions for VR, LT etc. It would be pretty ugly to break those statements over two lines.